### ISSUE TITLE
Not reserving RTCP Port

### ISSUE THREAD

**Issue by albertpv** [sippy/rtpproxy#81]
The RTPProxy is performing reservation for two RTP Ports on each session (one for RTP and one for RTCP). Is there any elegant way to disable the reservation of RTCP Port ? 

Our solution does not use/announce RTCP hence skipping that port would be appreciated.

**Comment by albertpv** [comment]
Any feedback?

**Comment by razvancrainea** [comment]
No, unfortunately there is no way to disable reservation of RTCP port. Perhaps you should flag this ticket as a feature request.

**Comment by albertpv** [comment]
Ok we can flag as feature request and in fact if i understand how to achieve this would have no problem to share with the community (maybe best would be as a define in makefile)

anyway in order to achieve this i tried removing any rtcp references in this file https://github.com/sippy/rtpproxy/blob/master/src/rtpp_session.c  and recompile the project.

But still sawing both rtp and rtcp ports reserved when a session is created. What i would be missing to achieve it?

**Comment by sobomax** [comment]
@albertpv could be easy, indeed. The RTCP port reservation is done deep inside the session creation code, however, so that you cannot decouple it from the RTP at higher level. Just because we need to allocate a consecutive pair (port, port+1). If we would allocate RTP session and initialize it first, and then do the same for RTCP and the second part fails we would have to tear down first and re-do. So you are looking at tweaking [create_twinlistener()](https://github.com/sippy/rtpproxy/blob/6681d45fdef40145d6a19731798e983d1bc04c27/src/rtpp_command.c#L105) function in the rtpp_command.c module.

My approach would be maybe making a dummy class with interface matching the rtpp_socket(), and construct that instead when i==1
